src/components/ClipboardButton/README-ru.md

```diff
@@ -30,22 +30,13 @@ LANDING_BLOCK-->
 
-`ClipboardButton` наследует свойства от `Button`, за исключением `href`, `component`, `target`, `rel`, `loading` и `children`.
-
-| Имя                | Описание                                                                    |                        Тип                        | Значение по умолчанию |
-| :----------------- | :-------------------------------------------------------------------------- | :-----------------------------------------------: | :-------------------: |
-| hasTooltip         | Включает или отключает отображение тултипа.                                 |                     `boolean`                     |        `true`         |
-| onCopy             | Обратный вызов после копирования:`(text: string, result: boolean) => void`. |                    `Function`                     |                       |
-| options            | Параметры копирования в буфер обмена.                                       | [CopyToClipboardOptions](#copytoclipboardoptions) |                       |
-| text               | Копируемый текст.                                                           |                     `string`                      |                       |
-| timeout            | Время до возврата состояния в норму после клика по кнопке.                  |                     `number`                      |        `1000`         |
-| tooltipInitialText | Текст, отображаемый перед копированием.                                     |                     `string`                      |       `"Copy"`        |
-| tooltipSuccessText | Текст, отображаемый после копирования.                                      |                     `string`                      |      `"Copied!"`      |
-
-### CopyToClipboardOptions
-
-| Имя     | Описание                                                                                                                                                                         |    Тип    | Значение по умолчанию |
-| ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-------: | :-------------------: |
-| debug   | Включает вывод в консоль.                                                                                                                                                        | `boolean` |        `false`        |
-| message | Сообщение-подсказка.                                                                                                                                                             | `string`  |                       |
-| format  | Устанавливает MIME-тип для копирования. Используйте `text/html` для копирования в формате HTML и `text/plain`, если при вставке в текстовый редактор не нужно наследовать стили. | `string`  |     `"text/html"`     |
+`ClipboardButton` наследует [свойства](../Button/README-ru.md#свойства) от `Button`.
```

Comment:
Don't we want to mention that it also inherits CopyToClipboardProps ? And doesn't it matter that not full scope of props is inherited?